cmd/plakar/plakar.go

```
@@ -150,6 +150,10 @@ func entryPoint() int {
 	}
 
 	// setup from default + override
+	if opt_cpuCount <= 0 {
+		fmt.Fprintf(os.Stderr, "%s: invalid -cpu value %d. Fallking back to 1.\n", flag.CommandLine.Name(), opt_cpuCount)
+		opt_cpuCount = 1
```

We should fail rather than defaulting to 1. Just like the check below when you are over NumCPU.

---

fixed